Test/Tests/Component/NodeManager.Tests.ps1

```diff
@@ -1,6 +1,7 @@
 Param (
     [Parameter(Mandatory=$false)] [string] $TestenvConfFile,
     [Parameter(Mandatory=$false)] [string] $LogDir = "pesterLogs",
+    [Parameter(Mandatory=$false)] [bool] $PrepareEnv = $true,
```

Comment:
Maybe we should somewhere in the README, that on the setup deployed with CAD, a user should specify `$PrepareEnv` to `$false` before running tests.

Reply:
Good point, forgot about it